app/features/send/send-provider.tsx

```diff
@@ -21,6 +22,7 @@ type Props = PropsWithChildren<{
 export const SendProvider = ({ initialAccount, children }: Props) => {
   const { mutateAsync: getInvoiceFromLud16 } = useGetInvoiceFromLud16();
   const { mutateAsync: createCashuSendQuote } = useCreateCashuSendQuote();
+  const { mutateAsync: getCashuSendSwapQuote } = useGetCashuSendSwapQuote();
```

Comment:
if we are calling this one get and one for send quote create I wonder if we should change the old one to also be called get? doesn't have to be done in this pr